lms/static/certificates/sass/_components.scss

```diff
@@ -562,3 +561,83 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: z-index(front);
+    background-color: palette(grayscale, trans); /*dim the background*/
+    vertical-align: middle;
```

Comment:
NIT: please sync up [property order](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order)